### ISSUE TITLE
[Research] OUI Compliance audit of the `visualizations` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4105]
Tracking issue for OUI compliance audit of the `visualizations` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
6.9K | [src/plugins/visualizations/public/wizard/_dialog.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/wizard/_dialog.scss) | 
1.5K | [src/plugins/visualizations/public/components/_visualization.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/components/_visualization.scss) | 
933 | [src/plugins/visualizations/public/embeddable/_embeddables.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/embeddable/_embeddables.scss) | 
232 | [src/plugins/visualizations/public/embeddable/_visualize_lab_disabled.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/embeddable/_visualize_lab_disabled.scss) | 
80 | [src/plugins/visualizations/public/index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/index.scss) | 
57 | [src/plugins/visualizations/public/embeddable/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/embeddable/_index.scss) | 
25 | [src/plugins/visualizations/public/components/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/components/_index.scss) | 
18 | [src/plugins/visualizations/public/wizard/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/visualizations/public/wizard/_index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.
